src/page_body/structure/blocks/page_block_tabs.pr

```diff
@@ -1,106 +1,107 @@
-{[ let block = context /]}
+{[ let block = context.block /]}
+{[ let pageId = context.pageId /]}
```

Comment:
note: @smcmullan-r7 I see that they are now namespacing this, and it's possible that we will have to namespace like this in our custom components (component status). could you please test that if you pull in THIS branch into our r7 documentation in supernova (master is selected by default so you'll need to manually swap to this branch) that our custom component still renders properly? if you take a screenshot of how you're consuming this branch and how it renders in published view please, that will be a proof of testing on this update. if it renders fine, we are good to merge this pr, otherwise we may need to update our custom component. hope that makes sense, let me know if we need to chat this out 👍

Reply:
this was tested, our components don't seem to break, but there is an issue with footer, that oif it persists, we'll create a ticket for.